src/main/scala/org/matthicks/media4s/video/info/MediaInfo.scala

```diff
@@ -35,21 +37,28 @@ case class MediaInfo(streams: List[MediaStream], format: MediaFormat) {
     }
   }
   def video: VideoInfo = videos.head
-  def audio: AudioInfo = audios.head
+  def audio: Option[AudioInfo] = audios.headOption
   def frames: Int = videos.headOption.map(_.fps * duration).map(_.toInt).getOrElse(0)
 
   def hasVideo: Boolean = videos.nonEmpty
   def hasAudio: Boolean = audios.nonEmpty
 
   override def toString: String = {
-    s"MediaInfo(duration: $duration, start: $start, video: $video, audio: $audio, frames: $frames)"
+    val audioString = audio.fold("No Audio")(audio => s"$audio")
```

Comment:
Perhaps:
`val audioString = audio.map(_.toString).getOrElse("No Audio")`
Might be a little clearer?

Reply:
Hi, 
Thanks. Is the code unclear or the comment?

Reply:
Personally, I think the code is unclear.  If you disagree, I won't hold-back the merge for this.

Reply:
Sure, I can make this more clear.